inflect-framework/client
# Fix: Non-mutating modal delete

- Divided state for population and visibility from create pipeline tab
- Consolidated logic for building of userCreatedPipeline object
- Added types for FrontendPipeline
- Revised Schema type

TODO:
- ability to add pipelines via post req to create_pipeline
- ability to pull/reference schema from server for use in create test event
- generate test event using pulled/referenced schema
- test test event after generation
- identify failed step on UI
- refactor TabbedModal to remove unnecessary state
- confirm ability to add/remove filters from list for a pipeline (Chris' ocal DB has no filters)
- remove Finalize tab
- Redefine Pipelines table to fit new DB schema
- Memoize cachable modal state
